client/Sheepshead/AndroidSheep/AndroidSheepGame.cs

```diff
@@ -0,0 +1,299 @@
+        }
+
+        private string Prompt(PromptType promptType, Dictionary<PromptData, object> data)
+        {
+            string prompt = "";
+            ITrick trick = null;
+            IPlayer player = null;
+            IPlayer picker = null;
+            switch (promptType)
+            {
+                case PromptType.CardsDealt:
+                    PlayerGraphicsDict = new Dictionary<IPlayer, AndroidPlayer>();
+                    var blinddata = (IBlind)data[PromptData.Blind];
+                    foreach (var addPlayer in _table.Players)
+                    {
+                        AndroidPlayer hostGraphics = new AndroidPlayer(addPlayer);
+                        PlayerGraphicsDict.Add(addPlayer, hostGraphics);
+                    }
+                    var blindLoc = 250;
+                    var blindIndex = 0;
+                    foreach (var addBlind in blinddata)
+                    {
+                        var cardtexture = GameContent.TextureDict[addBlind];
+                        var cardGraphics = new AndroidCard(cardtexture, addBlind)
+                        {
+                            IsBlind = true,
+                            Position = new Vector2(blindLoc, ScreenHeight / 4.0f)
+                        };
+                        BlindList[blindIndex] = cardGraphics;
+                        blindIndex++;
+                        blindLoc += 500;
+                    }
+                    foreach (var playerPair in PlayerGraphicsDict)
+                    {
+                        var playerGraphicsHand = playerPair;
+                        var playerHand = playerPair.Key.Hand;
+                        var screen = ScreenWidth / 2 - 500;
+                        foreach (var card in playerHand)
+                        {
+                            var cardtexture = GameContent.TextureDict[card];
+                            var cardGraphics = new AndroidCard(cardtexture, card)
+                            {
+                                Position = new Vector2(screen, ScreenHeight - 200)
+                            };
+                            playerGraphicsHand.Value.AddCardToHand(cardGraphics);
+                            screen += 150;
+                        }
+                    }
+                    ChangeState(new AndroidPreGameState(this, _graphics.GraphicsDevice, GameContent));
+                    State = StateType.PreGame;
+                    
+                    break;
+                    
+                case PromptType.Pick:
+                    ChangeState(_blindState);
+                    State = StateType.Blind;
+                    prompt = "yes"; 
+                    break;
+
+                case PromptType.PlayCard:
+                    State = StateType.Playing;                   
+                    picker = (IPlayer)data[PromptData.Picker];
+                    player = (IPlayer)data[PromptData.Player];
```

Comment:
I hated doing this over and over in console sheep. I wonder if we can just use trygetvalue() and just get them if they exist at the top of the switch. Not a priority to change (console sheep does it like this still).